examples/opf/tools/dataGenerationTool.py

```diff
@@ -22,8 +22,8 @@
 
```

Comment:
Done. Changed extension to `.pyw` to make it obvious that it is a GUI tool as well.